apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -36,6 +36,8 @@ export default class AssignmentSelector extends Component {
     primaryAssignmentIds: PropTypes.arrayOf(PropTypes.string).isRequired,
     chooseLaterOption: PropTypes.bool,
     dropdownStyle: PropTypes.object,
+    onChange: PropTypes.func,
```

Comment:
No, not yet.  I use it in my upcoming sections-dialog-redux PR.  In the interim I hope the test coverage is sufficient to not consider it dead code.